lib/active_admin/views/components/status_tag.rb

```diff
@@ -26,17 +27,12 @@ def default_class_name
       #   # => <span class='status_tag in_progress'>In Progress</span>
       #
       # @example
-      #   status_tag('active', :ok)
-      #   # => <span class='status_tag active ok'>Active</span>
-      #
-      # @example
-      #   status_tag('active', :ok, class: 'important', id: 'status_123', label: 'on')
-      #   # => <span class='status_tag active ok important' id='status_123'>on</span>
+      #   status_tag('active', class: 'important', id: 'status_123', label: 'on')
+      #   # => <span class='status_tag active important' id='status_123'>on</span>
       #
       def build(*args)
```

Comment:
Yes I agree. The API is clearer. @timoschilling you ok for me to make that change? I'll be sure to update the changelog above.